mmeds/spawn.py

```diff
@@ -404,7 +404,7 @@ def run(self):
                 self.logger.error(process)
                 # Whenever it's acceptable to move to Python 3.10 this needs to be turned into a switch statement
                 # If the watcher needs to shut down
-                if process == 'terminate':
+                if process == 'terminate' or process[0] == 'terminate':
```

Comment:
Why `process` or `process[0]`? Should be one or the other, the variable is either a string or an array.